rust/private/utils.bzl

```
@@ -179,6 +179,27 @@ def expand_locations(ctx, env, data):
     """
     return dict([(k, _expand_location(ctx, v, data)) for (k, v) in env.items()])
 
+def expand_list_element_locations(ctx, args, data):
+    """Performs location-macro expansion on a list of string values.
+
+    $(execroot ...) and $(location ...) are prefixed with ${pwd},
+    which process_wrapper and build_script_runner will expand at run time
+    to the absolute path.
+
+    See `expand_locations` for detailed documentation.
+
```

Nit: replace `expand_locations` with

```
[`expand_locations`](https://docs.bazel.build/versions/main/skylark/lib/ctx.html#expand_location)
```

Also above please.

---

```suggestion
See `expand_locations` for detailed documentation.
https://docs.bazel.build/versions/main/skylark/lib/ctx.html#expand_location
```

This isn't a rendered function so I think it'd be nicer to have the doc something that's easy to highlight, copy, and paste. But up to you two 😅